__tests__/pages/__snapshots__/curriculum.test.js.snap

```diff
@@ -2065,13 +2065,12 @@ exports[`Curriculum Page Should render with lessonStatus data 1`] = `
               A super simple introduction to help you get started!
             </p>
             <a
-              class="btn btn-sm btn-primary text-white float-end mb-2 me-2me-0"
+              class="btn btn-sm btn-primary text-white float-end mb-2 me-2 me-0"
+              data-testid="review-submissions-count"
               href="/review/js0"
             >
               Review 
-              <span>
-                0
-              </span>
+              <span />
                Submissions
```

Comment:
Is this supposed to not show "Review Submissions" if there are 0 submissions that need to be reviewed?

Reply:
No, something is wrong. I wonder if it's the conditions.